packages/techdocs-cli/src/commands/index.ts

```diff
@@ -223,6 +228,11 @@ export function registerCommands(program: Command) {
       '--no-docker',
       'Do not use Docker, run `mkdocs serve` in current user environment.',
     )
+    .option(
+      '--site-name',
+      'Name for site when using default MkDocs config',
+      'Table of Contents',
```

Comment:
Consider maybe using `basename` of the source directory instead of a hardcoded string?

Reply:
Can you clarify this one: do you mean just for the `serve:mkdocs` command or all of them? I haven't had a chance to test/confirm this but won't you end up with the random string that's generated as the temp folder if we use `basename`? Is that better than using the default of "Documentation Site"?

Reply:
Actually, that's a good point. I was thinking about manual preview but it's possible in some scenarios it's not a useful directory name. Ok, probably fine to skip this for now and use the catch all default site name.

Reply:
OK, sounds good 👍